### DISCUSSION TITLE
Google AMP, o que vocês acham?

### DISCUSSION THREAD

**Discussion by heliomsolivas** [frontendbr/forum#854] [Perguntas e Respostas]
Acredito que tudo depende do projeto, do cliente e tudo mais, mas recentemente li o artigo

https://www.theregister.co.uk/2017/05/19/open_source_insider_google_amp_bad_bad_bad/
http://desenvolvimentoparaweb.com/miscelanea/acabem-com-google-amp/

Vocês acham que isso seria uma boa "jogada"?

**Comment by felipefialho** [comment]
O segundo link está quebrado

**Comment by heliomsolivas** [comment]
@LFeh oxi que estranho aqui tá normal
